batch7crud-roll9/src/main/java/com/lftechnology/batch7crud/controller/StudentController.java

```diff
@@ -167,4 +189,63 @@ private void deleteProcess(HttpServletRequest request, HttpServletResponse respo
       show404(request, response);
     }
   }
+
+  private HashMap getParam(HttpServletRequest request) {
+    HashMap input = new HashMap();
+    input.put(ParameterConstant.FIRST_NAME, request.getParameter(ParameterConstant.FIRST_NAME));
+    input.put(ParameterConstant.MIDDLE_NAME, request.getParameter(ParameterConstant.MIDDLE_NAME));
+    input.put(ParameterConstant.LAST_NAME, request.getParameter(ParameterConstant.LAST_NAME));
+    input.put(ParameterConstant.ADDRESS, request.getParameter(ParameterConstant.ADDRESS));
+    input.put(ParameterConstant.GRADE, request.getParameter(ParameterConstant.GRADE));
+    return input;
+  }
+
+
+  private void setError(HttpServletRequest request, HttpServletResponse response, HashMap hm) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused method parameter(s) "response". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1172)